doc/LATENCY-OPTIMIZED-MODE.md

```
@@ -23,8 +23,8 @@ Should this BIOS option be unavailable or if there is a preference to change the
 
 |Platform	         |Script Type|	URL                                                                |
 |------------------|-----------|---------------------------------------------------------------------|
-|Linux/FreeBSD/UNIX|bash       | https://github.com/intel/pcm/blob/master/scripts/bhs-power-mode.sh  |
-|Windows	         |powershell | https://github.com/intel/pcm/blob/master/scripts/bhs-power-mode.ps1 |
+|Linux/FreeBSD/UNIX|bash       | https://github.com/khulnasoft/pcm/blob/master/scripts/bhs-power-mode.sh  |
+|Windows	         |powershell | https://github.com/khulnasoft/pcm/blob/master/scripts/bhs-power-mode.ps1 |
```

Markdownlint (0.35.0)

Column: 9
Hard tabs

(MD010, no-hard-tabs)